### PR TITLE
APP-1650: Test `applyRationCeiled` and `applyRatioFloored`

### PR DESCRIPTION
**Pull request by Michael-A-Heuer**
## Description

Task: [APP-1650](https://aragonassociation.atlassian.net/browse/APP-1650)

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [x] I have selected the correct base branch.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.
- [x] Any dependent changes have been merged and published in downstream modules.
- [x] I ran all tests with success and extended them if necessary.
- [x] I have updated the ``CHANGELOG.md`` file in the root folder.
- [x] I have tested my code on the test network.

[APP-1650]: https://aragonassociation.atlassian.net/browse/APP-1650?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: packages/contracts/contracts/utils/Ratio.sol [hunk 1]
```diff
@@ -11,10 +11,10 @@ uint256 constant RATIO_BASE = 10 ** 6;
 error RatioOutOfBounds(uint256 limit, uint256 actual);
 
 /// @notice Applies a ratio to a value and ceils the remainder.
-/// @param _value The value to which the ratio is applied .
-/// @param _ratio The ratio.
+/// @param _value The value to which the ratio is applied.
+/// @param _ratio The ratio that must be in the interval `[0, 10**6]`.
 /// @return result The resulting value.
-function applyRatioCeiled(uint256 _value, uint256 _ratio) pure returns (uint256 result) {
+function _applyRatioCeiled(uint256 _value, uint256 _ratio) pure returns (uint256 result) {
```
Comment (brickpop) [inline packages/contracts/contracts/utils/Ratio.sol:17]: Why don't we leave these function names as they were?  We could simply use `testApplyRatioCeiled()` on the new test contract, relaying to `applyRatioCeiled()`

(suggestion, not important)

File: packages/contracts/test/utils/ratio.ts [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+      const tooLargeValue = ethers.constants.MaxUint256.div(RATIO_BASE).add(1);
+      await expect(
+        ratio.applyRatioCeiled(tooLargeValue, RATIO_BASE)
+      ).to.revertedWithPanic(0x11);
+
+      await expect(ratio.applyRatioCeiled(tooLargeValue.sub(1), RATIO_BASE)).to
+        .not.be.reverted;
+    });
+
+    it('does not ceil for division without reminder', async () => {
```
Comment (brickpop) [inline packages/contracts/test/utils/ratio.ts:46]:
```suggestion
    it('does not ceil for division without remainder', async () => {
```

File: packages/contracts/contracts/utils/Ratio.sol (hunk 1 above)
Comment (Michael-A-Heuer) [reply packages/contracts/contracts/utils/Ratio.sol:17]: File-level functions will be included as internal functions in the contract. Our convention is to add an `_` as a prefix so I thought it is better to make clear that this is not a public function.
We do the same with `_auth` from `auth.sol`

**Review by brickpop — DISMISSED** [review]

**Review by mathewmeconry — CHANGES_REQUESTED** [review]
you know what you need to fix ^^

**Review by mathewmeconry — APPROVED** [review]

**Review by brickpop — APPROVED** [review]

**Pull request merged**